backtrace-library/src/main/java/backtraceio/library/BacktraceClient.java

```diff
@@ -75,7 +131,20 @@ public BacktraceClient(Context context, BacktraceCredentials credentials,
      */
     public BacktraceClient(Context context, BacktraceCredentials credentials,
                            Database database) {
-        this(context, credentials, database, null);
+        this(context, credentials, database, (Map<String, Object>) null);
```

Comment:
Does it make sense to make a cast that will return null instead of passing an empty dictionary to avoid confusion? Later you need to check anyway if the dictionary exists.

Reply:
Will do, thank you!